pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -78,11 +78,13 @@ layout: wide_index
 
   $(document).ready(function() {
     // Send through some values that the JavaScript will need.
+    // As of 2024 we are no longer showing the "Sort by" dropdown per line 87.
+    // See https://github.com/code-dot-org/code-dot-org/pull/60728 for more details.
     var tutorialExplorerManager = new window.TutorialExplorerManager({
       tutorials: #{raw tutorials_contents.to_json},
       locale: "#{locale_code}",
       roboticsButtonUrl: "#{resolve_url('/learn/robotics')}",
-      showSortDropdown: #{DCDO.get('learn_show_sort_dropdown', true)},
+      showSortDropdown: false,
```

Comment:
I assume this then defaults to sorting by displayweight, not popularity, right?

Reply:
**Update:** all hoc_modes are showing the same sort order so I think we're good to go! I suspect this is because the `displayweight_i` and `popularity_i` columns are now identical, which we'll maintain going forward.